examples/containers/multi/helpers/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Change 
+ARG couchbase_server_image=couchbase/server:enterprise-6.6.3
+
+# Build the exporter
+FROM golang:1.17.2 as go_build
+RUN git clone https://github.com/couchbase/couchbase-exporter.git /opt/couchbase-exporter
```

Comment:
In fact I think this is all overtaken by events now, @markspolakovs definitely has another version and we should just use that rather than duplicate it. Create a symlink if you have to but you should be able to do a `docker build -f <dir1>/Dockerfile <dir2 context>`

Reply:
https://github.com/couchbaselabs/observability/blob/main/testing/resources/containers/cb-with-exporter.Dockerfile is where my version lives now, for reference

Reply:
Yeah, just use that so we don't have to remember to change multiple things next time the server image changes.

Reply:
@markspolakovs since I'll be pointing to yours could you add `ARG GOLANG_VERSION=1.17.2` as a build arg like you wanted?